### PR TITLE
Port Arc60h hotswap PCB

### PR DESCRIPTION
**Pull request by stanrc85**
Port for Arc60h hotswap PCB, this version uses a unique json compared to the solder version.

### CODE REVIEW

File: keyboards/vertex/arc60h/keymaps/vial/keymap.c [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      break;
+    case 3:
+      rgblight_setrgb (0xFF, 0x00, 0x00);
+      break;
+    default: //  for any other layers, or the default layer
+      rgblight_setrgb (0x00, 0x00, 0x00);
+      break;
+    }
+  return state;
+}*/
```
Comment (lesshonor) [inline keyboards/vertex/arc60h/keymaps/vial/keymap.c]:
```suggestion
```
Always turn off RGB when the keyboard starts? Don't think most people would want that.

File: keyboards/vertex/arc60h/keymaps/vial/rules.mk [hunk 2]
```diff
@@ -0,0 +1,7 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+VIAL_ENABLE = yes
+COMBO_ENABLE = no
+KEY_OVERRIDE_ENABLE = no
```
Comment (lesshonor) [inline keyboards/vertex/arc60h/keymaps/vial/rules.mk]:
```suggestion
```
Do you really need to disable these?

File: keyboards/vertex/arc60h/keymaps/vial/vial.json [hunk 3]
```diff
@@ -0,0 +1,234 @@
+{
+  "name": "arc60h",
+  "vendorId": "0x7374",
+  "productId": "0x9770",
+  "lighting": "qmk_rgblight",
+  "matrix": {"rows": 5, "cols": 15},
+"layouts": {
+"labels":[
+      "Backspace",
```
Comment (lesshonor) [inline keyboards/vertex/arc60h/keymaps/vial/vial.json]:
```suggestion
      "Split Backspace",
```

File: keyboards/vertex/arc60h/keymaps/vial/config.h [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+#define VIAL_KEYBOARD_UID {0xE9, 0xF0, 0x2B, 0x71, 0x8D, 0x00, 0x29, 0x40}
+
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 2 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 13 }
+#define DYNAMIC_KEYMAP_LAYER_COUNT 4
```
Comment (lesshonor) [inline keyboards/vertex/arc60h/keymaps/vial/config.h]:
```suggestion
```
This is the default, so you don't really need to mention it.

**Review by lesshonor — CHANGES_REQUESTED** [review]
...This one you might want to mark as draft until the rest of the keyboard makes it into Vial. Since it was added to upstream QMK less than a month ago, that might be a while.

File: keyboards/vertex/arc60h/keymaps/vial/keymap.c (hunk 1 above)
Comment (stanrc85) [reply keyboards/vertex/arc60h/keymaps/vial/keymap.c]: I was building this for users on the Arc60 discord, I had intended to remove it sorry about that.

**Comment by stanrc85** [comment]
all changes have been addressed.  I'll mark as draft for now until the changes from QMK are merged.  Thanks

**Comment by stanrc85** [comment]
This should be ready for review and merge now that the PCB is added from latest QMK merge.

**Pull request merged**